test/es-module/test-esm-repl-imports.js

```diff
@@ -1,19 +1,27 @@
 'use strict';
+
 const { mustCall } = require('../common');
-const assert = require('assert');
 const fixtures = require('../common/fixtures');
-const { spawn } = require('child_process');
+const assert = require('node:assert');
+const { spawn } = require('node:child_process');
+const { execPath } = require('node:process');
+const { describe, it } = require('node:test');
 
-const child = spawn(process.execPath, [
-  '--interactive',
-], {
-  cwd: fixtures.path('es-modules', 'pkgimports'),
-});
 
-child.stdin.end(
-  'try{require("#test");await import("#test")}catch{process.exit(-1)}'
-);
+describe('ESM: REPL runs', { concurrency: true }, () => {
+  it(() => {
+    const child = spawn(execPath, [
+      '--interactive',
+    ], {
+      cwd: fixtures.path('es-modules', 'pkgimports'),
+    });
 
-child.on('exit', mustCall((code) => {
-  assert.strictEqual(code, 0);
-}));
+    child.stdin.end(
+      'try{require("#test");await import("#test")}catch{process.exit(-1)}'
+    );
+
+    child.on('exit', mustCall((code) => {
+      assert.strictEqual(code, 0);
```

Comment:
That makes the `mustCall` is a bit redundant here, but that can't hurt.
```suggestion
      assert.strictEqual(code, 0);
      done();
```